### ISSUE TITLE
Option to make the applications grid appear directly after pressing the super key (instead of the overview).

### ISSUE THREAD

**Issue by Amin394T** [home-sweet-gnome/dash-to-panel#2001]
Basically an option that will invert the order of appearance of the apps grid and the overview:
- pressing Super will show the apps grid,
- maybe double-pressing Super or Super+ALT will show the overview.

Just another feature to change GNOME's workflow to the classic workflow, without having to add extra extensions (like ArcMenu).

**Comment by natewind** [comment]
Yeah, it would be great to not need a separate extension just for that. The overview is basically useless with Dash to Panel, since the running apps are already on the panel.
